routes/search.js

```diff
@@ -0,0 +1,51 @@
+const express = require('express');
+const router = express.Router();
+const axios = require("axios")
+const jquery = require("jquery")
+const db = require('../dbcon.js');
+
+// Book object
+function Book(bookId, title, author, genre, isbn, imgUrl) {
+    
+    this.bookId = bookId;
+    this.title = title;
+    this.author = author;
+    this.genre = genre;
+    this.isbn = isbn;
+    this.imgUrl = imgUrl;
+}
+
+// Queries
+
+const selectAllAvailableBooks = `SELECT Books.id AS bookId, title AS title, author AS author, genre AS genre, isbn AS isbn, imgUrl AS imgUrl
```

Comment:
Need to get copies available count by either doing a subquery on user/books to find the number of copies currently available. Also, instead of doing a join on transactions since we don't really need data from that table I'd just do something like WHERE UserID NOT IN (SELECT UserBookId FROM Transactions) or something like that. Typically want to reserve joins for when we actually need data from the adjoining table or need it to create a subsequent join.